src/gradualizer_bin.erl

```diff
@@ -7,7 +7,7 @@
 %% of the elements. The returned type is a normalized bitstring type.
 -spec compute_type(ExprOrPat) -> gradualizer_type:abstract_type()
         when ExprOrPat :: {bin, _, _},
-             ExprOrPat :: erl_parse:abstract_expr().
+             ExprOrPat :: gradualizer_type:abstract_expr() | gradualizer_type:af_pattern().
```

Comment:
I'm not sure explicitly added `af_pattern()` is necessary.

Reply:
I though pattern is a subtype of expression, syntactically.

Reply:
semantically they are different, that's why I felt the difference, but indeed syntactically they are not.
I even asked gradualizer... so I will remove this modification
```
1> typechecker:subtype(typelib:parse_type("gradualizer_type:af_pattern()"),typelib:parse_type("gradualizer_type:abstract_expr()"), {tenv, #{}, #{}}).
{true, ...}
```